source/_docs/energy/gas.markdown

```diff
@@ -23,3 +23,66 @@ We have worked with creator [Marcel Zuidwijk](https://www.zuidwijk.com) to devel
 
 ![Photo of SlimmeLezer attached to a smart electricity meter](/images/docs/energy/slimmelezer.jpg)
 
+#### Reading the meter via a pulse counter
+
+Some Gas Meters, similar to Electric Meters, have an LED that will flash whenever a fixed amount of energy passes through it. By monitoring the time between flashes it’s possible to determine the energy consumption.
+
+Like Electric Meters, you can use Home Assistant Glow, an open source solution powered by ESPHome’s pulse meter sensor. You put it on top of the activity LED of your gas meter and it will bring your consumption into Home Assistant.
+
+#### Reading the meter via a magnetic pulse counter
+
+Older meters are unlikely to have an LED that flashes. In fact, many don’t even have LED displays, but instead use multiple analogue dials to document usage in units of 1s, 10s, 100s, 1000s, 10,000s, etc. However, some of these meters can still use a pulse counter to determine usage.
+
+As these analogue dials often rotated based on a magnet spinning due to gas usage, a pulse counter could be constructed to count the magnetic “pulses”, or each time the magnetic made a completed rotation. In fact, many older meters were specifically designed with this in mind, and the single digit dial is set up so that its complete rotation corresponds to the magnetic complete rotation. Some meters even had a spot under this dial to monitor magnetic pulses. 
```

Comment:
```suggestion
As these analog dials often rotated based on a magnet spinning due to gas usage, a pulse counter could be constructed to count the magnetic “pulses”, or each time the magnetic made a completed rotation. In fact, many older meters were specifically designed with this in mind, and the single-digit dial is set up so that its complete rotation corresponds to the magnetic complete rotation. Some meters even had a spot under this dial to monitor magnetic pulses. 
```